mpi-proxy-split/mpi-wrappers/mpi_group_wrappers.cpp

```diff
@@ -50,6 +50,23 @@ USER_DEFINED_WRAPPER(int, Comm_group, (MPI_Comm) comm, (MPI_Group *) group)
   return retval;
 }
 
+int
+MPI_Comm_internal_vgroup(MPI_Comm comm, MPI_Group *group)
```

Comment:
I suggest changing ``MPI_Comm_internal_vgroup` to `MPI_Comm_internal_virt_group`.  Long names are fine if it makes things eaiser for the reader of the code, and we don't use the 'v' prefix convention elsewhere.